src/Lean/Server/Snapshots.lean

```diff
@@ -23,8 +23,10 @@ open Elab
 /-- The data associated with a snapshot is different depending on whether
 it was produced from the header or from a command. -/
 inductive SnapshotData where
-  | headerData : Environment → MessageLog → Options → SnapshotData
+  | headerData : Command.State → SnapshotData
   | cmdData : Command.State → SnapshotData
+  -- NOTE(WN): these carry the same dat but we leave open the future
+  -- possibility of more snapshot variants. For example, within tactic blocks.
```

Comment:
At this point we can effectively guarantee that we will not solve tactic incrementality with server snapshots, I think.

Reply:
Fair enough! I will change this to just `Command.State`.